kaggle_plankton_tutorial.py

```diff
@@ -32,8 +27,8 @@
 warnings.filterwarnings("ignore")
 
 # get the classnames from the directory structure
-directory_names = list(set(glob.glob(os.path.join("/Users/applepei/Documents/Work/Kaggle/Plankton2015","train", "*"))\
- ).difference(set(glob.glob(os.path.join("/Users/applepei/Documents/Work/Kaggle/Plankton2015","train","*.*")))))
+directory_names = list(set(glob.glob(os.path.join('''Path removed''',"train", "*"))\
```

Comment:
HINT, you can make it to read from configuration file, and by default, it's using the current directory.  I think that's the convention.